controllers/provisioners/eks/update.go

```diff
@@ -116,6 +116,12 @@ func (ctx *EksInstanceGroupContext) UpdateScalingGroup() error {
 		ctx.Log.Info("updated scaling group tags", "instancegroup", instanceGroup.GetName(), "scalinggroup", asgName)
 	}
 
+	if ctx.SuspendProcessesUpdateNeeded() {
```

Comment:
One more question.. 
Since inside of `SetSuspendProcesses` or `SetResumeProcesses` we use:
```
if len(processesToSuspend) == 0 {
		return nil
	}
```
Does that mean we can get rid of the predicate and allow `UpdateScalingProcesses` to always be called?

Reply:
Hi @eytan-avisror ,
Yes, that's right. Post the code refactor this check is no longer required. I have removed the check and corresponding method.

Thank you.